### PR TITLE
Add Nausea Meds (Pepto-Bismol)

### PR DESCRIPTION
**Pull request by dissociativity**
```SUMMARY: Content "adds medicine to help nausea."```

**Reason for PR:**
Player spends far too much time retching over everyday things happening with weak stomach even with healthy diet resulting in crafting taking for ages and even deaths from moves lost, nausea and vomitious should still be quite bad, but the traits are now far less over-tuned given irradation and other effects.
I've found even minor irradation causes constant nausea on even a weak stomach'd character.

**Alternatives:**
Alternatives would be tweaking causes of nausea, making overall health affect how bad nausea is, or making lower amounts of radiation not cause constant retching at least til 4 or 5 full counts.

**Scenario:**
Player is irradiated a very small amount (1-3 counts, keeping in mind that it doesn't particularly harm the player til around 10, at least given I'm playing with mutation radiation turned off)
weak stomach results in regular retching and vomiting on a healthy diet with complete nutrition.

This is an everyday occurrence in PK rebalance which makes heavy use of radiation where it's presence in the base game is far lower.

### CODE REVIEW

**Comment by dissociativity** [comment]
done

**Comment by mlangsdorf** [comment]
Yes, you completely ignored the instructions included in the PR on how to submit a PR.  You need to have a properly formatted summary line, which you do not have.

**Comment by dissociativity** [comment]
okay, formatting now all prettied up and removed irrelevant details

**Comment by kevingranade** [comment]
Specifically you need a line that says
SUMMARY: Balance "Your summary line here."

**Comment by dissociativity** [comment]
Jenkins Rebuild

**Comment by dissociativity** [comment]
Done

**Comment by KorGgenT** [comment]
note: you've linted files that weren't previously linted. This is generally a thing that should have its own PR, so i suggest reverting the lint and adding whatever you added to the files.

**Comment by dissociativity** [comment]
Reverted Lint

**Comment by dissociativity** [comment]
well that was a pain, done

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -1334,6 +1334,27 @@
     "use_action": "WEAK_ANTIBIOTIC",
     "flags": "NPC_SAFE"
   },
+  {
+    "id": "pepto",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "heartburn medicine",
+    "description": "Creamy pink heartburn syrup that soothes upset stomachs and quells vomitus urges; with a twist off cap that doubles as a dosage cup.  Repeated doses may cause drowsiness.",
```
Comment (Rivet-the-Zombie) [inline data/json/items/comestibles/med.json]: vomitus -> vomitous

The former means 'matter that has been vomited' while the latter means 'nauseating'.

File: data/json/effects.json [hunk 2]
```diff
@@ -1902,6 +1902,31 @@
             "per_mod": [2]
         }
     },
+  {
+    "type": "effect_type",
+    "id": "tummy_tablet",
+    "name": [ "Soothed Stomach." ],
```
Comment (Night-Pryanik) [inline data/json/effects.json]:
```suggestion
    "name": [ "Soothed stomach" ],
```

**Comment by Night-Pryanik** [comment]
I'd remove the painkiller effect. If I understand it correctly, painkiller effect in this medicine simulates neutralizing pain in the stomach, but our current mechanics means that consuming this medicine will remove the pain in broken leg, for example.

**Comment by dissociativity** [comment]
That's fair, done

**Comment by dissociativity** [comment]
Alright, I think I addressed everything that anyone has asked for thus far!

File: data/json/items/comestibles/med.json [hunk 3]
```diff
@@ -1334,6 +1334,27 @@
     "use_action": "WEAK_ANTIBIOTIC",
     "flags": "NPC_SAFE"
   },
+  {
+    "id": "pepto",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "heartburn medicine",
+    "description": "Creamy pink heartburn syrup that soothes upset stomachs and quells vomitus urges; with a twist off cap that doubles as a dosage cup.",
```
Comment (Rivet-the-Zombie) [inline data/json/items/comestibles/med.json]:
```suggestion
    "name": [ "Creamy pink heartburn syrup that soothes upset stomachs and quells vomitous urges; with a twist off cap that doubles as a dosage cup." ],
```

**Pull request merged**